app/src/pages/PageProjects.re

```diff
@@ -76,9 +78,14 @@ let make = () => {
     {
       switch (simple) {
       | Error(err) =>
-        <div className="error">
-          {"ERROR: " ++ err##message |> React.string}
-        </div>
+        dispatch(
+          AlertsAction(
+            StoreAlerts.Show(
+              Alert.{severity: Alert.Error, message: err##message},
```

Comment:
I agree, we should invest some time to figure out how we guide the user in case there are some problems. We should provide the user with a good explanation of what happened and what to do next to resolve the issue. Additionally we should think about what tone we use in the messaging (formal vs informal/friendly).

But I think this should happen at a later stage when we already have nailed down the user flows then we'll be able to see what the actual error cases are and provide the necessary guidance.